<launch>
    <node pkg="phoxi_camera" type="phoxi_camera" name="phoxi_camera">
        <rosparam file="$(find phoxi_camera)/config/phoxi_camera.yaml" command="load"/>
    </node>
</launch>
